### PR TITLE
[#14388] Re-enable bug reporting

### LINKED ISSUE #14388: Can't report bug from profile and on shaking device
# Bug Report

## Problem 
No pop-up when shaking the device and no redirect to email on pressing the `Submit by email` button in the `Need help?` section.

https://user-images.githubusercontent.com/67952253/202404934-89806e00-8360-4a89-adf8-d88085dbf34d.mp4

#### Expected behavior
Can report a bug

#### Actual behavior
`Submit a bug` и `Shake phone` features don't work

### Reproduction

1. Open Status
2. Shake your device

or

1. Open Profile > Need help? > Submit a bug
2. Enter a description and steps 
3. Tap `Submit by email with logs archive`

### Additional Information

- Status version: nightly 17/11
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by rasom**
fix #14388

status: ready

### CODE REVIEW

File: src/status_im2/setup/events.cljs [hunk 1]
```diff
@@ -10,7 +10,8 @@
             [status-im.multiaccounts.login.core :as multiaccounts.login]
             [status-im.native-module.core :as status]
             [status-im.utils.keychain.core :as keychain]
-            [status-im2.navigation.events :as navigation]))
+            [status-im2.navigation.events :as navigation]
+            status-im.utils.logging.core))
```
Comment (flexsurfer) [inline src/status_im2/setup/events.cljs]: its better to add to `status-im.events` because its not migrated yet

**Review by flexsurfer — APPROVED** [review]

File: src/status_im2/setup/events.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im2/setup/events.cljs]: ok, but why did you remove it from `status-im.events` in the first place?

Comment (flexsurfer) [reply src/status_im2/setup/events.cljs]: by mistake :)

Comment (rasom) [reply src/status_im2/setup/events.cljs]: got it, thanks

Comment (flexsurfer) [reply src/status_im2/setup/events.cljs]: thank you

**Pull request merged**